how-to-guides/re-run-failed-training/notebooks/re_run_failed_training.ipynb

```diff
@@ -0,0 +1,540 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "THIzFFk81nme"
+   },
+   "source": [
+    "# How to re-run failed training"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Introduction\n",
+    "When you are executing a model training script that’s being tracked in Neptune and it fails in the middle, you can easily re-run it with the same metadata such as hyperparameters, data, and code version.\n",
+    "\n",
+    "By the end of this guide, you will learn how to re-open a failed Neptune run to fetch the metadata needed to re-run it and log all metadata from the model training, validation, or testing to a new run, to save results you didn’t get from the failed run.\n",
+    "\n",
+    "[See this example in Neptune](https://app.neptune.ai/common/showroom/e/SHOW-3695)\n",
+    "\n",
+    "[![image](https://files.gitbook.com/v0/b/gitbook-x-prod.appspot.com/o/spaces%2F-MT0sYKbymfLAAtTq4-t%2Fuploads%2Fcgj0RmMm89ejwZKSZfXK%2Fimage.png?alt=media&token=c0314a39-5e8d-465c-9752-a688bd207b91)](https://app.neptune.ai/common/showroom/e/SHOW-3695)\n",
```

Comment:
All files should be hosted in wordpress. Please coordinate with @Patrycja-J

Reply:
Let's just remove this for now, as the screenshot is outdated anyway.

Reply:
Got it.

I will remove all the screenshots